packages/vue/src/provide/createProvide.ts

```diff
@@ -38,7 +38,7 @@ function createScope<T extends string>(scopeName: T, createScopeDeps: CreateScop
    * --------------------------------------------------------------------------------------------- */
 
   function createProvide<ProvideValueType extends object | null>(
-    rootComponentName: T,
+    rootComponentName: string,
```

Comment:
why change string `T` -> `string` ?

Reply:
in the original it has a ```string``` type

[link](https://github.com/oku-ui/primitives/pull/531#discussion_r1510651446)

AND

fix type error

[link](https://github.com/oku-ui/primitives/blob/b860fd4667d7987d9982562ad00dd09ebda707d8/packages/vue/src/accordion/props.ts#L94)